server/integration/testsuite/pom.xml

```diff
@@ -1052,6 +1059,7 @@
                             <groups>${groups.security}</groups>
                             <systemPropertyVariables>
                                 <arquillian.launch>${suite.security.arquillian}</arquillian.launch>
+                                <infinispan.module-suffix>${infinispan.module-suffix}-security</infinispan.module-suffix>
```

Comment:
Why do we need those module suffixes?

Reply:
It's so that the module's test suite gets a unique log file name (https://github.com/infinispan/infinispan/blob/40918fc5540ec17afb2316f052577602065161fd/etc/log4j2-trace.xml#L15). Nowadays we keep the directory structure for the artifacts, so we could get away without it, but I think it still helps a bit.